lib/MTT/Test/Build/Trivial.pm

```diff
@@ -56,6 +58,12 @@ sub Build {
                        "trivial_tests_cflags");
     my $fflags = Value($ini, $config->{full_section_name}, 
                        "trivial_tests_fflags");
+    my $f77flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f77flags");
+    my $f90flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f90flags");
+    my $f08flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f08flags");
```

Comment:
I was just being symmetric with $mpif77 $mpif90 and $mpif08
shall I replace these too ? or is it fine to be somehow inconsistent ?